apps/android/boot.js

```diff
@@ -197,6 +197,21 @@
           lastSteps = steps;
         }, event.int*1000);
       },
+      // {t:"actFetch", ts:int}
+      "actFetch": function() {
```

Comment:
Should maybe be actlast, as the existing code in Gadgetbridge that I missed.

Reply:
Thanks! Yes, I think `actlast` would be good as the code's already in Gadgetbridge for that now...